test/fixtures/get-ledger-full.json

```diff
@@ -0,0 +1,64 @@
+{
+    "stateHash": "2C23D15B6B549123FB351E4B5CDE81C564318EB845449CD43C3EA7953C4DB452",
```

Comment:
Wasn't this fixture here before? Why is it being re-added here?

Reply:
I added this was added by mistake because I saw a ledger-full in a different PR's test, but I couldn't find the file it in the fixture folder (I guess I just missed it). Removed the duplicate!